src/clustering/extreme_vals.jl

```diff
@@ -92,7 +92,19 @@ function simple_extr_val_ident(clust_data::ClustData,
                                extremum::String="max",
                                peak_def::String="absolute",
                                consecutive_periods::Int=1)
-  data=clust_data.data[data_type]
+  # all attribute-node pairs in clust_data
+  data_types = [k for k in keys(clust_data.data)]
+  attribute_nodes = data_types[occursin.(data_type, data_types)]
+  if isempty(attribute_nodes)
```

Comment:
Instead of leaving the if statement open, we could just return the error and end the if statement after line 99. What do you think?